### ISSUE TITLE
'bld_ao_bielec_integrals_in_map_$THREAD_NUM' is being used without being defined

### ISSUE THREAD

**Issue by TApplencourt** [LCPQ/quantum_package#46]
When compiling without `openmp` flag and launch SCF binary 

```
AO map initialized
 Providing the AO integrals
forrtl: severe (193): Run-Time Check Failure. The variable 'bld_ao_bielec_integrals_in_map_$THREAD_NUM' is being used without being defined
Image              PC                Routine            Line        Source             
SCF                00000000005BE803  provide_ao_bielec         133  ao_bi_integrals.irp.F90
SCF                000000000049687A  bld_ao_bi_elec_in         616  Fock_matrix.irp.F90
SCF                000000000049B268  provide_fock_matr         519  Fock_matrix.irp.F90
SCF                000000000049E310  provide_fock_matr         734  Fock_matrix.irp.F90
SCF                000000000056D53F  huckel_guess_              41  huckel.irp.F90
SCF                000000000040377C  MAIN__                    107  SCF.irp.F90
SCF                00000000004032E6  Unknown               Unknown  Unknown
libc.so.6          00007F1A949C3EC5  Unknown               Unknown  Unknown
SCF                00000000004031D9  Unknown               Unknown  Unknown
```

**Comment by TApplencourt** [comment]
Maybe check : EZFIO folder

**Comment by scemama** [comment]
Solved with irpf90 version 1.6
